packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+:root {
+  --token-accordion-item-padding: 16px;
+  --token-accordion-item-padding-right: 12px;
```

Comment:
Is the intention to move these to the design tokens package? If not, we seem to be using SASS variables for other components, potentially a topic for our engineering sync to decide the approach forward.

Reply:
+1 in converting these to Sass variables

Reply:
I will convert to Sass variables, thanks.